CONTRIBUTING.md

```diff
@@ -0,0 +1,249 @@
+# Contributors Guide
+
+Interested in helping out the IOOS QARTOD?
```

Comment:
QARTOD is 1 module of `ioos_qc`. There are others QA/QC standards implemented here.

```suggestion
Interested in helping out the ioos_qc?
```